crates/db_views/src/comment_view.rs

```diff
@@ -372,7 +372,7 @@ impl<'a> CommentQueryBuilder<'a> {
       SortType::Hot | SortType::Active => query
         .order_by(hot_rank(comment_aggregates::score, comment_aggregates::published).desc())
         .then_order_by(comment_aggregates::published.desc()),
-      SortType::New => query.order_by(comment::published.desc()),
+      SortType::New | SortType::Forum => query.order_by(comment::published.desc()),
```

Comment:
This won't be shown on the front end for anything but Posts, but I think its a good idea to be explicit about these nonsensical sorts, rather than using a catch all. Maybe in the future we should split out two different SortTypes.